src/content/reference/cluster-definition/_index.md

```diff
@@ -133,9 +133,13 @@ Coming from v4, you might want to understand how v5 is different from v4:
 - `owner`: Name of the owner organization.
 - `name`: Friendly name of the cluster. If not specified, a name will be generated.
 - `release_version`: Allows to select a specific release version. The value must be the semver version number of an active release. To get information on all available releases, use the [`gsctl list releases`](/reference/gsctl/list-releases/) command.
-- `master`:
-  - `availability_zone`: Name of the availability zone to use for the master node. If not set, one will be assigned randomly.
+- `master_nodes`: Settings regarding the Kubernetes master nodes.
+  - `high_availability`: Where supported, this is `true` by default, which means that the cluster will have three master nodes. Supported on AWS since release v{{% first_aws_ha_masters_version %}}. Set this to `false` to have only one master node in the cluster (not recommended for production clusters).
```

Comment:
I'm a bit torn about "not recommended for production" because it sounds like what we've been doing before is really not safe. Maybe rather say that HA is "recommended for production"?

Reply:
I am writing "recommended only for test clusters". Naturally, with the introduction of HA, our recommendation changes.